app/controllers/register_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RegisterController < Devise::RegistrationsController
+
+  private
+
+  def sign_up_params
```

Comment:
these things being used anywhere? or does devise look for these functions explicitly?